app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/utils/BeaconPlacer.kt

```diff
@@ -0,0 +1,98 @@
+package ch.epfl.cs311.wanderwave.ui.components.utils
+
+import ch.epfl.cs311.wanderwave.model.data.Beacon
+import ch.epfl.cs311.wanderwave.model.data.Location
+import com.google.android.gms.maps.model.LatLng
+import kotlin.math.*
+import kotlin.random.Random
+
+private const val EARTH_RADIUS_M = 6371000.0
+private const val BEACON_RADIUS = 1000.0
+private const val BEACON_COUNT = 20
+private const val NUMBER_ITERATION = 5
+
+fun placeBeaconsRandomly(beacons: List<Beacon>, location: LatLng): List<Beacon> {
+  var finalBeacons = mutableListOf<Beacon>()
```

Comment:
If you could add some comments it would be quite helpful I think, it'd make it easier to understand what happens